src/check.ts

```diff
@@ -259,6 +259,7 @@ export interface CommonIssue {
    * Only available if the server runs in Targets mode.
    */
   scoring?: GoalScoring;
+  aiRephraseHint: string;
```

Comment:
@dj-delima Shouldn't this be optional field as it was introduced newly in API, platforms with older version won't have this `aiRephraseHint` field.